extensions/plugins/system/redcore/redcore.xml

```diff
@@ -122,6 +122,16 @@
                     <option value="0">JNO</option>
                 </field>
                 <field
+                        name="stateful_webservice"
+                        class="btn-group"
+                        type="radio"
+                        default="0"
```

Comment:
the old behavior is 1. So I would prefer if it stay that way or the extensions using webservices will get sudden change of behavior.